### ISSUE TITLE
Ensure blog posts can be posted to the new site

### ISSUE THREAD

**Issue by omerbensaadon** [knative/docs#3540]
## Description: 
Since we want to limit the scope of our work in this first release of the MkDocs site, we have opted to keep our Blog in the existing Hugo template and wire it together with the MkDocs prototype. 

See: https://knative.dev/blog/

For the current process, see: https://github.com/knative/docs/blob/3af9db6781370363e333bd00d0043d407f3d8142/help/contributor/new-blog/_index.md 

---
## Story
```
As Carlos the Community Manager
I want to post a new blog onto the new Knative Site
So that I can create release and feature announcements, articles and social events for Danny. 
```

## Acceptance Criteria
`I can post a new blog`
```
GIVEN I have a new blog to post 
WHEN I commit a .md file to any of the folders in /docs/blog/ 
THEN I see that file as a blogpost on the Knative Site
```

`MkDocs ignores blog posts`
```
GIVEN I have committed a .md file to any of the folders in /docs/blog/
WHEN I build the site with MkDocs 
THEN MkDocs ignores these files 
```

## Known Issues and Considerations: 
- **This story can be delivered, infrastructure-wise, in a number of ways.** However, there may be a way to implement this which would be less of a headache to the current docs writers and blog Editorial team. It would be great if we could explore a few options before implementing.

**Comment by omerbensaadon** [comment]
I think this is fixed in #3558 since it leverages the existing site repo

**Comment by omerbensaadon** [comment]
Closed by #3558
